zktest/src/archive/test2/B85-ZK-3717.zul

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+B85-ZK-3717.zul
+
+	Purpose:
+		
+	Description:
+		
+	History:
+		Tue Aug 16 11:52:15 CST 2017, Created by bobpeng
+
+Copyright (C) 2017 Potix Corporation. All Rights Reserved.
+
+-->
+<zk xmlns:n="native">
+	<zscript><![CDATA[
+        public void print(event, a){
+        	String str = "Event: " + event + a.getValue();
+            System.out.println(str);
```

Comment:
Printing to console is hard to debug. Use Clients.log to print to browsers instead.